### PR TITLE
Sw mobile home page bug

### LINKED ISSUE #72: SearchNEU logo obscures query input on mobile
Screenshot_20210121-122151_Fennec

### PR DESCRIPTION
**Pull request by sebwittr**
# Purpose

###### A brief description of the purpose of the PR's changes for the project.
Fixed some visual bugs that occurred on the Mobile Home Page.

<br>

# Tickets

###### A link to any tickets this PR is associated with on Trello.

-https://trello.com/c/fL3mPBAc/270-fix-mobile-homepage

# Contributors

###### Anyone who contributed to this PR for future reference.

-Sebastian and Zachar

# Feature List

###### A list of more in-depth and technical changes, additions, deletions, and fixes this PR provides.

-
-
-

# Notes

###### A list of miscellaneous notes for this pull request, such as whether we need to add something later for production, or that it was time boxed, etc.

<br>

# Checklist

- [x] Filled out PR template :wink:
- [ ] Approved by designers
- [ ] Is passing linting checks
- [ ] Is passing tsc
- [ ] Is passing existing tests
- [ ] Has documentation and comments in code
- [ ] Has test coverage for code
- [ ] Will have clear squash commit message

<br>
@sandboxnu/searchneu

### CODE REVIEW

File: pages/[campus]/[termId].tsx [hunk 1]
```diff
@@ -35,12 +35,7 @@ export default function Home(): ReactElement {
 
   const alertBanners = Object.values(alertBannersData) as [AlertBannerData];
 
-  // On mobile only show the logo and the github corner if there are no results and the search box is not focused (the virtual keyboard is not on the screen).
   let containerClassnames = 'home-container';
```
Comment (hankewyczz) [inline pages/[campus]/[termId].tsx]: This can be `const`, since we no longer re-assign it

**Review by hankewyczz — COMMENTED** [review]
Code looks fine, just change the `let` so the linter stops complaining. 

Also, I don't see two columns for the notification - maybe there's a commit that wasn't pushed? I thought the desired behavior was a two-column flexbox (https://www.w3schools.com/css/css3_flexbox_responsive.asp)

image

Basically, the 'X' and text would be side by side until the text overflows, at which point the 'X' goes below and center of the text. Right now, the X is just off to the side and below, which looks off. 

For tickets like these (ie. visual changes), it's very helpful to attach screenshots of the changed behavior - a before/after.

**Comment by sebwittr** [comment]
I changed it so the banner would become a column if on mobile. I attached images of the old banner and the new banner, so we can see the difference.
Old Banner:
previous-banner

New Banner:
new-banner

**Review by hankewyczz — APPROVED** [review]

**Comment by hankewyczz** [comment]
#154 #72

**Pull request merged**